lib/ansible/modules/cloud/heroku/heroku_collaborator.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: heroku_collaborator
+short_description: "Add or delete app collaborators on Heroku"
+version_added: "2.6"
+description:
+  - Manages collaborators for Heroku apps.
+  - If set to C(present) and heroku user is already collaborator, then do nothing.
+  - If set to C(present) and heroku user is not collaborator, then add user to app.
+  - If set to C(absent) and heroku user is collaborator, then delete user from app.
+author:
+  - Marcel Arns <marcel.arns@moneymeets.com>
+requirements:
+  - heroku3
+options:
+  api_key:
+    description:
+      - Heroku API key
+    required: false
+  apps:
+    description:
+      - List of Heroku App names
+    required: true
+  suppress_invitation:
+    description:
+      - Suppress email invitation when creating collaborator
+    type: bool
+    required: false
+    default: "False"
```

Comment:
not a blocker, but it looks better in the docs if you use (yes/no) for type bool.